components/automate-cli/cmd/chef-automate/pullAndGenerateConfig.go

```diff
@@ -232,7 +232,8 @@ func (p *PullConfigsImpl) setExceptionIps(ips []string) {
 	p.exceptionIps = ips
```

Comment:
As the current code, For 5 node cluster : for success case if we delete the automate node. 
config still contain the chef-server-ips : which will fail the cert-rotate, config patch command. 
That needs to be address in this PR.

Reply:
we have discussed it, this case should not occur